src/frontend.md

```diff
@@ -229,8 +229,18 @@ export function fakeTicket() {}
 
 1. If some color is repeated several times in code, extract it to theme.
 
+## Disposition of components
+
+1. Place a component into UI Kit if it is added to the designer's shared UI-kit or matches wih MUI components library. [Why MUI?](#ui-kit)
+
+1. Expose a component from MF if it represents a domain of microfrontend and not being common.
+
+1. Place a component into `websome-kit`/`agent-kit` if a component depends on Osome SDK and for some reason it is not desirable to serve it through MF.
+
 ## UI-kit
 
+1. Use `MUI` as the reference for `Component API`. Use **identical** naming for components and props to have a **roadmap** how to scale further.
```

Comment:
how about adding a reason why we need it?

for example:
`It's needed for having consistent naming and keeping components universal.`